notebooks/5_CUR.ipynb

```diff
@@ -162,10 +162,12 @@
     "For the $j^{th}$ column, the importance score is given by\n",
     "\n",
     "\\begin{equation}\n",
-    "\\pi_j = \\sum_i^k (U_C)_{ij}^2,\n",
+    "\\pi_j = \\sum_j^k (U_C)_{ij}^2,\n",
```

Comment:
This is not correct. It is the sum over `i` corresponding to column `j`. If we sum over `j` we're finding the importance of `i`.

Reply:
I will fix it.